pkg/agent/plugin/svidstore/utils.go

```diff
@@ -0,0 +1,94 @@
+package svidstore
+
+import (
+	"crypto/x509"
+	"fmt"
+	"strings"
+
+	"github.com/spiffe/spire/pkg/common/pemutil"
+	"github.com/spiffe/spire/pkg/common/x509util"
+	svidstorev1 "github.com/spiffe/spire/proto/spire/plugin/agent/svidstore/v1"
+)
+
+type Secret struct {
```

Comment:
The name `Secret` feels like specific to the aws_secretsmanager plugin. Maybe something like `SVID`?